frontend/src/libs/components/link/link.tsx

```diff
@@ -1,15 +1,29 @@
 import { NavLink } from 'react-router-dom';
 
 import { type AppRoute } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
 import { type ValueOf } from '#libs/types/types.js';
 
+import styles from './styles.module.scss';
+
 type Properties = {
   to: ValueOf<typeof AppRoute>;
   children: React.ReactNode;
+  className?: string | undefined;
 };
 
-const Link: React.FC<Properties> = ({ children, to }: Properties) => (
-  <NavLink to={to}>{children}</NavLink>
-);
+const Link: React.FC<Properties> = ({
+  children,
+  to,
+  className = '',
+}: Properties) => {
```

Comment:
```suggestion
}) => {
```
FC<Properties> already uses the types for the props

Reply:
yes, but i had this lint error: 'className' is missing in props validationeslint[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)
(parameter) className: string

Reply:
<img width="821" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/fa79a98d-dd48-477a-8c85-e674eb46402f">

Reply:
@ossavemu  disable this rule in frontend eslintrc (FYI you can change configs only when one of the coaches ask you)

Reply:
done :)

Reply:
The problem is with the eslintrc.yml, for some reason when modifying it this error appears. I have tried many ways but I can't figure out how to commit after editing it.
I then restated the type in the FC link props.

Reply:
You should add your updated eslintrc to git stage

Reply:
Don't forget to stage your files

Reply:
Solved!